docs/usage.md

```diff
@@ -428,7 +429,8 @@ spec:
       value: minioadmin
     - name: AWS_SECRET_ACCESS_KEY
       value: minioadmin
-
+    - name: AWS_REGION
+      value: us-east-1
     # bucketName is required.  Other fields are optional.
```

Comment:
Please update this comment. When the `backendType` is `s3`(default). `region` is required.
```
    # bucketName is required.  Other fields are optional.
```

Reply:
@masa213f 
I don't well understand for backup of MOCO. 
but, I tried to make explanation for this s3 case config.
please review for is it right.

Reply:
@masa213f 
I updated for your this request.